gameserver/GameBoard/GameBoard.ts

```diff
@@ -30,7 +32,7 @@ export class GameBoard {
 
     public giveCards() {
         this.players.forEach(player => {
-            player.setCards(this.deck.getDeck().splice(0,9));
+            player.setCards(this.deck.getDeck().splice(0,this.deck.getDeck().length / playerNumber));
```

Comment:
Is not gonna work, you need to compute the number of cards to draw outside the loop